daemon/resources/man/razer-service.8

```diff
@@ -28,16 +30,13 @@ Stop any existing daemon first, if one is running.
 Gracefully stop the existing daemon.
 .TP
 \fB--config\fR=\fIconfig_file\fR
-Specifies the location of the config file. If this is not provided it will default to \fB$HOME\fR/.razer-service/razer.conf and create it if needed from the example config.
+Specifies the location of the config file. If this is not provided it will default to \fB$HOME\fR/.config/razer-service/razer.conf and create it if needed from the example config.
```

Comment:
Uhm if you could create me the text / PR a better text it would be way easier for me as formulating (good) text isn't really easy for me.